jvm-packages/checkstyle.xml

```diff
@@ -16,8 +16,8 @@
   -->
 
 <!DOCTYPE module PUBLIC
-          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
-          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+    "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+    "https://checkstyle.org/dtds/configuration_1_3.dtd">
```

Comment:
just question, what happened to them?

Reply:
Frankly I do not know.
It may be worth reporting to the owner of puppycrawl.com